config/clusterpermissions/cluster_role.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I don't see how we are adding the ClusterRoleBinding to the CSV.  When I generated the bundle on your branch, I didn't see anything about clusterPermissions in the CSV.  Also, shouldn't there be an update to the CSV with clusterPermissions in it?

Reply:
I understand this now.  Thanks for clarifying.